native/packages/react-native-bpk-component-banner-alert/src/style/index.android.js

```diff
@@ -0,0 +1,65 @@
+import {
+  colorGray50,
+  spacingBase,
+  spacingMd,
+  spacingSm,
+  spacingLg,
+  lineHeightXs,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import commonStyle from './style.common';
+
+const androidStyles = Object.assign({},
```

Comment:
the whole approach here is to use the base style already created and override what's needed, `Stylesheet.create` is used in the main component so this is a plain object at this point